<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>191</width>
      <height>147</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="expCntLabel">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
        </font>
       </property>
       <property name="text">
        <string>실험회차</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nameLEdit">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
        </font>
       </property>
       <property name="text">
        <string>InTaekJung</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="birthLabel">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
        </font>
       </property>
       <property name="text">
        <string>생년월일</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="expCntLEdit">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
        </font>
       </property>
       <property name="text">
        <string>피험자명</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="submitBtn">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
        </font>
       </property>
       <property name="text">
        <string>문제풀이 시작</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="birthLEdit">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
        </font>
       </property>
       <property name="text">
        <string extracomment="ex. 901114">901114</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>나눔스퀘어_ac</family>
        </font>
       </property>
       <property name="text">
        <string>실험유형</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="expTypeCBox">
       <item>
        <property name="text">
         <string>Pre-Test</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Post-Test</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔스퀘어_ac</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>[미프] 온라인 강의 시청 실험</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
